scripts/user_functions.py

```diff
@@ -185,6 +156,35 @@ def eval_expr(expr: str) -> str:
     return f"{eval(expr)}"
 
 
+def extract_keywords_from(parts: List[str]) -> Dict[str, str]:
+    """
+    Given a list of strings, extract strings containing an equal sign ("=").
+
+    Return a *defaultdict(str)* with key=value extracted from the original list.
+
+    The left part of the sign is used a as the dict key and the right part as the value value.
```

Comment:
The left part of the sign is used as the dict key and the right part as the value